Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

In file included from src/unicode.c:21:
src/definitions.h:48:15: error: expected identifier
typedef enum {false, true} bool;
              ^
/usr/include/stdbool.h:36:15: note: expanded from macro 'false'
#define false   0
                ^

--- src/definitions.h.orig	2020-07-18 06:57:36 UTC
+++ src/definitions.h
@@ -43,13 +43,7 @@
 
 /*==== Misc definitions ====*/
 /* Define a bool type if not already defined (C++ and C99 do)*/
-#if !(defined(__cplusplus) || (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 19990601L))
-/*@-incondefs@*/
-typedef enum {false, true} bool;
-/*@+incondefs@*/
-#elif defined(__STDC_VERSION__) && __STDC_VERSION__ >= 19990601L
 #include <stdbool.h>
-#endif
 
 /*==== Configuration definitions ====*/
 #ifndef NO_STRDUP
